CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 <!-- SPDX-License-Identifier: Apache-2.0 -->
 
+## Unreleased
+
+### Added
+
+* Add `BitcrowdEcto.Schema.safe_cast_enum_member/3`, a function that safely converts a string to the member of an enum based on reflection on the given schema.
```

Comment:
How about `safe_cast_to_enum_member`? Or `cast_to_enum_member`? (What's "safe" about the conversion?)

Reply:
`cast_to_enum_member` 👍 

safe because it's verifying that the string is part of the enum without using `String.to_atom` or `String.to_existing_atom` 🤷

Reply:
maybe even `to_enum_member`. After all `to_string` and `to_atom` also do not have a `cast` in the name :)